site/get-started.markdown

```diff
@@ -0,0 +1,118 @@
+---
+title: Get Started
+page: get-started
+isGettingStarted: true
+---
+
+# Get started
+
+Welcome, new Haskeller! Read on to quickly set up your Haskell development environment, execute your first lines of code, and get directions for further learning.
+
+## Content
+  - [Set up a Haskell development environment](#set-up-a-haskell-development-environment)
+    - [GHCup: universal installer](#ghcup-universal-installer)
+    - [Editor](#editor)
+  - [Running your first lines of code](#running-your-first-lines-of-code)
+  - [Writing your first Haskell program](#writing-your-first-haskell-program)
+  - [Participate in the community](#participate-in-the-community)
+  - [Next steps](#next-steps)
+
+## Set up a Haskell development environment
+
+A complete Haskell development environment consists of the Haskell toolchain (compiler, language server, build tool) and an editor with good Haskell support. The quickest way to get this set up is to:
+
+1. Use **GHCup** to install and manage the Haskell toolchain.
+2. Use [**VSCode**](https://code.visualstudio.com/) as the editor, with the Haskell extension installed.
```

Comment:
How would you feel about linking to the free [VSCodium](https://vscodium.com/) builds instead of the non-free version?

Reply:
I understand the motivation, but I don't think that is beneficial for the goal of reducing friction to get new people to learn Haskell. Those that care about open source will probably reach out vscodium anyway, while for those that don't, this probably isn't the best moment to introduce it -> we are already introducing Haskell which is complex enough.

If this will be a stopper for merging, I can change it to VSCodium, but I think it sends the wrong message to newcomers to Haskell.